ts-eks/test/klotho.yaml

```diff
@@ -0,0 +1,63 @@
+app: ts-microservices
+provider: aws
+path: .
+out_dir: compiled
+defaults:
+    execution_unit:
+        type: lambda
+        pulumi_params_by_type:
+            fargate:
+                cpu: 256
+                memory: 512
+            lambda:
+                memorySize: 512
+                timeout: 180
+    static_unit:
+        type: s3
+        pulumi_params_by_type:
+            s3:
+                cloudFrontEnabled: true
+    expose:
+        type: apigateway
+    persist:
+        kv:
+            type: dynamodb
+        fs:
+            type: s3
+        secret:
+            type: s3
+        orm:
+            type: rds_postgres
+            pulumi_params_by_type:
+                rds_postgres:
+                    allocatedStorage: 20
+                    engineVersion: "13.7"
+                    instanceClass: db.t4g.micro
+                    skipFinalSnapshot: true
+        redis:
+            type: elasticache
+            pulumi_params_by_type:
+                elasticache:
+                    nodeType: cache.t3.micro
+                    numCacheNodes: 1
+    pubsub:
+        type: sns
+execution_units:
+    microsrv-api:
+        type: eks
+        pulumi_params:
+            nodeType: node
+    microsrv-users:
+        type: eks
+        pulumi_params:
+            nodeType: node
+    sequelize-main:
```

Comment:
fwiw this sequelize-main isn't present in the base klotho.yaml file (non test one)

Not sure if it's intended or not

Reply:
Also if it's meant to be fargate/node

Reply:
the base klotho.yaml got pushed up by accident. want it to be fargate so we mix and match